src/MapWithSearch/LocationSearch.js

```diff
@@ -94,15 +94,17 @@ class LocationSearchInput extends React.Component {
               <InputGroup.Append>
                 <Button
                   onClick={this.props.onGeolocation}
-                  variant="outline-secondary"
+                  variant="light"
+                  className="border-1"
                 >
-                  <FontAwesomeIcon icon={faCrosshairs} />
+                  <i className="fal fa-crosshairs"></i>
```

Comment:
this change will need to be reflected in imports on this page.

Reply:
Check again?